src/extension/background-script/connectors/lnd.ts

```diff
@@ -223,11 +227,75 @@ class Lnd implements Connector {
     });
   };
 
-  getTransactions = () => {
-    return this.request("GET", "/v1/payments", undefined, {
-      transactions: [],
-    });
-  };
+  async getInvoices(): Promise<GetInvoicesResponse> {
+    const data = await this.request<{
```

Comment:
we miss pagination here.
`index_offset` `num_max_invoices`

there is also a `reversed` parameter which we probably want to use instead of the reverse() in JS.

if we set `reversed` to true, then we should get the newest ones and probably also can ignore pagination for now.

Reply:
it seems it is somewhat working for me. 
I get new transactions BUT still need the reverse() ? 

<img width="454" alt="image" src="https://user-images.githubusercontent.com/318/180187211-f32f56ba-5aae-41b1-bce8-bbfeef2f09ab.png">

Reply:
As discussed. Reversing the reversed result...